Source/WebCore/layout/formattingContexts/inline/InlineItemsBuilder.cpp

```diff
@@ -114,6 +114,58 @@ void InlineItemsBuilder::build(InlineItemPosition startPosition)
     };
     adjustInlineContentCacheWithNewInlineItems();
 
+    auto adjustElementBoundaryTextSpacings = [&] {
```

Comment:
this function is getting too long and this is certainly not core functionality, could you move it to a dedicated function, please?